crates/router/src/services.rs

```diff
@@ -9,7 +9,7 @@ use std::sync::{atomic, Arc};
 use redis_interface::{errors::RedisError, PubSubInterface};
 
 pub use self::api::*;
-#[cfg(feature = "basilisk")]
+// #[cfg(feature = "basilisk")]
```

Comment:
remove commented code, why we are moving out from basilisk feature?

Reply:
+1